code/go/0chain.net/smartcontract/dbs/event/updater.go

```diff
@@ -0,0 +1,112 @@
+package event
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/0chain/common/core/logging"
+	"github.com/lib/pq"
+	"gorm.io/gorm"
+)
+
+const SetTemplate = "%v = t.%v"
+const ExprTemplate = "%v = %v"
+const UnnestTemplate = "unnest(?::%v[]) AS %v"
+const UpdateTemplate = "UPDATE %v SET"
+const WhereTemplate = "WHERE %v.%v = t.%v"
+const QueryTemplate = "%v %v FROM (SELECT %v) AS t %v"
+
+var typeToSQL = map[reflect.Type]string{
+	reflect.TypeOf([]string{}):  "text",
+	reflect.TypeOf([]int64{}):   "bigint",
```

Comment:
Add `[]uint64` would make it easier for 'Stake' value updating and also for those uin64 field such as the 'Downtime' for health check.

Reply:
we will add it in the next issue, we need to merge it asap